docs/_lab/01-installing-bolt/index.md

```diff
@@ -4,16 +4,41 @@ difficulty: Basic
 time: Approximately 10 minutes
 ---
 
-In this exercise you will install Bolt so you can get started with Bolt Tasks.
+In this exercise you will install Bolt and create a Boltdir so you can get started with Bolt.
 
-# Installation Details
+## Installation Details
 
 Bolt is packaged for the major operating systems. Please refer to the [installation documentation](https://puppet.com/docs/bolt/latest/bolt_installing.html) to install Bolt for the OS you are using. 
 
-**Note** For this lab and for most use cases it is recommended that bolt is NOT installed as a Ruby Gem. This is because optional (but highly useful) supporting modules are only included in packages and must be installed manually when using the Gem.    
+**Note** For this lab and for most use cases it is recommended that bolt is NOT installed as a Ruby Gem. This is because optional (but highly useful) supporting modules are only included in packages and must be installed manually when using the Gem.
 
-# Next steps
+## Creating a Boltdir
 
-Now that you have Bolt installed you can move on to:
+By default `$HOME/.puppetlabs/bolt/` is the base directory for user-supplied data such as the configuration and inventory files. It is effectively the default `Boltdir`. 
+You may find it useful to maintain a project specific `Boltdir`. When you commit a `Boltdir` to a project you can share Bolt configuration and code between users.
```

Comment:
```suggestion
You may find it useful to maintain a project specific Bolt project directory. When you commit a Bolt project directory to a project you can share Bolt configuration and code between users.
```

Reply:
Sorry I know this reads *terribly* but I think using the same phrase is important :(